AlignedCollectionViewFlowLayout/Classes/AlignedCollectionViewFlowLayout.swift

```diff
@@ -230,8 +233,43 @@ public class AlignedCollectionViewFlowLayout: UICollectionViewFlowLayout {
         lineFrame.size.width = lineWidth
         return super.layoutAttributesForElements(in: lineFrame) ?? []
     }
-    
-    /// Copmutes the alignment axis with which to align the items represented by the `layoutAttributes` objects vertically.
+
+    /// Determines the total width of layout attributes objects for all items including minimumLineSpacing.
+    ///
+    /// - Parameter layoutAttributes: The layout attributes for an entire line.
+    /// - Returns: The real content width for all the layout attributes objects representing all items in the same line.
+    fileprivate func contentWidthForLine(with layoutAttributes: [UICollectionViewLayoutAttributes]) -> CGFloat {
```

Comment:
I think we should consider another name for this function because `contentWidth` is already used for a computed property with a different meaning that describes the _full_ available width of the collection view. (Or we change the name of the `contentWidth` property, maybe even a better idea...) Otherwise these two things might be easily confused.